React/Views/ScrollView/RCTScrollView.m

```diff
@@ -538,21 +523,6 @@ - (RCTUIView *)contentView // TODO(macOS ISS#3536887)
   return _scrollView.documentView;
 }
 
-- (BOOL)canBecomeFirstResponder
-{
-  return [_scrollView canBecomeFirstResponder];
```

Comment:
Does removing these overrides affect its subviews' getting focus?

Reply:
If by "it" you mean RCTScrollView's subview _scrollView, AFAIK it never got focus. The outer RCTScrollView always got focus, and managed keyboard events, etc. This code made it look like `_scrollView` got focus, but if checked in Xcode via lldb,  `[[[NSApp windows] objectAtIndex:0] firstResponder]` (AKA, the first responder of the main window) was always the outer RCTScrollView and not the inner _scrollView.

Reply:
Oh, another thing. When I experimented with making the subview (_scrollView) get focus instead of the outer view (RCTScrollView), the key view loop broke, not sure why :/. That's part of why I elected to keep logic as is with focus only going to the outer view.